src/Umbraco.Web.UI.NetCore/Startup.cs

```diff
@@ -42,12 +51,27 @@ public Startup(IWebHostEnvironment webHostEnvironment, IConfiguration config)
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddUmbracoSqlCeSupport();
```

Comment:
This should be part of our CoreComposer, i'll have a look at fixing this up

Reply:
I understand why some of this seems to need to be setup before the rest, mostly to do with the SqlMainDom but that only requires that this is registered `DbProviderFactories.RegisterFactory(Core.Constants.DbProviderNames.SqlServer, SqlClientFactory.Instance);` which could easily be part of it's own construction. It just doesn't seem right that we need to boot things before we boot the core which is also why we need to pre-add things for the integrations tests. Maybe you have other thoughts about this one but i'll cleanup things how i think it could work and maybe you can let me know if you disagree.